datadog_checks_base/datadog_checks/base/checks/win/wmi/sampler.py

```diff
@@ -323,35 +369,6 @@ def _format_property_values(self, previous, current):
 
         return formatted_wmi_object
 
-    def get_connection(self):
-        """
-        Create a new WMI connection
-        """
-        self.logger.debug(
-            u"Connecting to WMI server (host=%s, namespace=%s, provider=%s, username=%s).",
-            self.host,
-            self.namespace,
-            self.provider,
-            self.username,
-        )
-
-        # Initialize COM for the current thread
-        # WARNING: any python COM object (locator, connection, etc) created in a thread
-        # shouldn't be used in other threads (can lead to memory/handle leaks if done
-        # without a deep knowledge of COM's threading model). Because of this and given
-        # that we run each query in its own thread, we don't cache connections
```

Comment:
Can we make sure to keep this historical comment, so we don't lose that piece of info (it seems useful)?

Reply:
Done - moved to _query_sample_loop where the CoInitialize is done